lib/cli.ex

```diff
@@ -22,12 +24,22 @@ defmodule TicTacToe.CLI do
       """
   end
 
+  def get_number_of_players() do
+    IO.gets("Please enter the number of players (1 or 2)")
+      |> String.trim
+      |> String.to_integer
+  end
+
   def get_player_name(marker, gets \\ &IO.gets/1) do
     gets.("Player #{marker}, please enter your name:\n")
       |> String.trim()
   end
 
-  def get_player_move(player, gets \\ &IO.gets/1) do
+  def get_player_move(%Player{computer: computer}, board, _gets) when computer == true do
```

Comment:
This method is a tough one to place. On one hand, it seems like it should belong to the UI given the context; on the other, it knows about a higher level module (i.e, ComputerPlayer, and its associated method).

This might be something that should belong on the Player objects themselves.